build-rpi-raspbian/first-boot.sh

```diff
@@ -5,6 +5,8 @@ exec &> >(tee -a "/boot/first-boot.log")
 
 timestamp() { date +"%F_%T_%Z"; }
 
+sh /boot/webif.sh start
```

Comment:
Please move below function `fail_inprogress` and add a comment or log line.

Reply:
i moved it below `Storing configuration`, so we can get the hostname out of openhabian.conf